.github/workflows/wheel_linux_x86_64.yml

```diff
@@ -216,5 +216,5 @@ jobs:
         uses: pypa/gh-action-pypi-publish@release/v1
         with:
           user: __token__
-          password: ${{ secrets.TEST_PYPI_API_TOKEN }}
+          password: ${{ matrix.pl_backend == "lightning_qubit" && secrets.TEST_PYPI_API_TOKEN || secrets.TEST_PYPI_LKOKKOS_TOKEN }}
```

Comment:
Can you also update changelog as this PR extends the supported devices in Test PyPI?